Lib/test/test_unittest/test_runner.py

```diff
@@ -106,6 +107,7 @@ def cleanup2(*args, **kwargs):
         self.assertTrue(test.doCleanups())
         self.assertEqual(cleanups, [(2, (), {}), (1, (1, 2, 3), dict(four='hello', five='goodbye'))])
 
+    @force_not_colorized
```

Comment:
we have to apply this decorator to so many methods that I almost wonder if it's worth having a custom metaclass that automatically adds it to each method on the class... or we could not use the `@force_not_colorized` decorator, and instead duplicate the logic in `setUp` and `tearDown` methods on the `TestCleanup` class here.

Neither feels ideal; it might be that what you have now is in fact best!